include/deal.II/grid/grid_tools.h

```diff
@@ -1361,7 +1361,8 @@ namespace GridTools
       std::vector<unsigned int> recv_ptrs;
 
       /**
-       * Distribute quadrature points on found intersections and construct
+       * Distribute structdim*n_points_1D quadrature points on found
```

Comment:
Is this right? Shouldn't it be `n_points_1d ^ structdim`?

Reply:
Thank you for spotting, this is what I wanted to write. I checked the Code and `QGaussSimplex()` does not always have `n_points_1d ^ structdim` points. So I rephrased the documentation.